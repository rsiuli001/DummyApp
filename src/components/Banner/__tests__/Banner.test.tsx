import React from 'react';
import { View, Text } from 'react-native';
import { shallow } from 'enzyme';
import { Banner } from '../..';

const bannerProps = {
  deviceType: true,
  styles: {},
};

describe('Banner', () => {
  describe('render banner', () => {
    const wrapper = shallow(
      <Banner deviceType={bannerProps.deviceType} styles={bannerProps.styles} />
    );

    it('should render the banner view', () => {
      expect(wrapper).toMatchSnapshot();
      expect(wrapper.find(View).length).toEqual(1);
    });

    it('should render the banner text', () => {
      expect(wrapper).toMatchSnapshot();
      expect(wrapper.find(Text).length).toEqual(1);
    });
  });
});
