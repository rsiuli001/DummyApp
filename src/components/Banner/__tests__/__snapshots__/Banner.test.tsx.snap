// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner render banner should render the banner text 1`] = `ShallowWrapper {}`;

exports[`Banner render banner should render the banner view 1`] = `ShallowWrapper {}`;
